dmutils/flask.py

```diff
@@ -32,3 +33,24 @@
 timed_render_template_string.__doc__ = """
     See ``timed_render_template``, only for ``render_template_string``.
 """
+
+
+class DMGzipMiddleware(Gzip):
+    compress_by_default: bool
+
+    def __init__(self, *args, compress_by_default: bool = False, **kwargs):
```

Comment:
I think setting a default parameter has the same effect as using kwargs.setdefault (and is more readable).

```suggestion
    def __init__(self, *args, compress_by_default: bool = False, minimum_size = 8192, **kwargs):
```

Reply:
But then it opens the possibility of forgetting to add it back to the kwargs in the supercall...